x-pack/test/alerting_api_integration/observability/helpers/alerting_api_helper.ts

```diff
@@ -64,5 +64,8 @@ export async function createRule<Params = ThresholdParams>({
       rule_type_id: ruleTypeId,
       actions,
     });
+  if (body.statusCode && body.statusCode !== 200) {
+    throw new Error(body.message);
+  }
```

Comment:
```suggestion
  expect(body.statusCode).eql(200, body.message);
```

this is how i usually use it. if it fails, it will also print messages needed.

Reply:
I've tried it locally but got this error:
![image](https://github.com/elastic/kibana/assets/12370520/8f5e1900-c442-48a5-a769-c3b375e26fdd)

Reply:
`import expect from '@kbn/expect';`
you need to add import